examples/checkout-book.yaml

```diff
@@ -46,7 +46,7 @@
     resourceQuantity:
       qudt:unit: unit:Number
       qudt:numericValue: 1
-    time:before: 2019-05-17T18:00:00-5:00
+    due: 2019-05-17T18:00:00-5:00
 
```

Comment:
We don't have "due" in VF. This should be:
---
before: 2019-05-17T18:00:00-5:00
---
this is "vf:before", not "time:before".